src/wolfgame.py

```diff
@@ -7354,7 +7367,7 @@ def fgame(cli, nick, chan, rest):
 
             if gamemode not in var.GAME_MODES.keys() - var.DISABLED_GAMEMODES:
                 gamemode = gamemode.split()[0]
-                gamemode, _ = complete_match(gamemode, var.GAME_MODES.keys() - var.DISABLED_GAMEMODES)
+                gamemode = complete_match(gamemode, var.GAME_MODES.keys() - var.DISABLED_GAMEMODES)
```

Comment:
You're getting a list as a response - didn't you want `complete_one_match` instead?